include/librealsense2/rsutil.h

```diff
@@ -213,4 +213,11 @@ static void rs2_project_color_pixel_to_depth_pixel(float to_pixel[2],
     }
 }
 
+
+static rs2_sensor_mode rs2_resolution_from_width_height(int width, int height)
```

Comment:
I'd rather not have this inpublic API - it is too much SKU-dependent and also does not provide an added value